src/proposals/mips/mip-b09/mip-b09.sol

```diff
@@ -0,0 +1,132 @@
+import {Configs} from "@proposals/Configs.sol";
+import {Proposal} from "@proposals/proposalTypes/Proposal.sol";
+import {Addresses} from "@proposals/Addresses.sol";
+import {JumpRateModel} from "@protocol/IRModels/JumpRateModel.sol";
+import {TimelockProposal} from "@proposals/proposalTypes/TimelockProposal.sol";
+import {CrossChainProposal} from "@proposals/proposalTypes/CrossChainProposal.sol";
+import {Comptroller} from "@protocol/Comptroller.sol";
+
+contract mipb09 is Proposal, CrossChainProposal, Configs {
+    string public constant name = "MIP-b09";
+    uint256 public constant timestampsPerYear = 60 * 60 * 24 * 365;
+    uint256 public constant SCALE = 1e18;
+
+    uint256 public constant WETH_PREVIOUS_CF = 0.8e18;
+    uint256 public constant WETH_NEW_CF = 0.81e18;
+
+    uint256 public constant USDC_PREVIOUS_CF = 0.82e18;
+    uint256 public constant USDC_NEW_CF = 0.83e18;
+
+    uint256 public constant cbETH_PREVIOUS_CF = 0.75e18;
+    uint256 public constant cbETH_NEW_CF = 0.76e18;
+
+    constructor() {
+        bytes memory proposalDescription = abi.encodePacked(
+            vm.readFile("./src/proposals/mips/mip-b09/MIP-B09.md")
+        );
+        _setProposalDescription(proposalDescription);
+    }
+
+    function _validateCF(
+        Addresses addresses,
+        address tokenAddress,
+        uint256 collateralFactor
+    ) internal {
+        address unitrollerAddress = addresses.getAddress("UNITROLLER");
+        Comptroller unitroller = Comptroller(unitrollerAddress);
+
+        (bool listed, uint256 collateralFactorMantissa) = unitroller.markets(
+            tokenAddress
+        );
+
+        assertTrue(listed);
+
+        assertEq(
+            collateralFactorMantissa,
+            collateralFactor,
+            "collateral factor validation failed"
+        );
+    }
+
+    function deploy(Addresses addresses, address) public override {}
+
+    function afterDeploy(Addresses addresses, address) public override {}
+
+    function afterDeploySetup(Addresses addresses) public override {}
+
+    function build(Addresses addresses) public override {
+        address unitrollerAddress = addresses.getAddress("UNITROLLER");
+
+        // =========== ETH CF Update ============
+
+        // Add update action
+        _pushCrossChainAction(
+            unitrollerAddress,
+            abi.encodeWithSignature(
+                "_setCollateralFactor(address,uint256)",
+                addresses.getAddress("MOONWELL_WETH"),
+                WETH_NEW_CF
+            ),
+            "Set collateral factor for ETH"
+        );
+
+        // =========== USDC CF Update ============
+
+        // Add update action
+        _pushCrossChainAction(
+            unitrollerAddress,
+            abi.encodeWithSignature(
+                "_setCollateralFactor(address,uint256)",
+                addresses.getAddress("MOONWELL_USDC"),
+                USDC_NEW_CF
+            ),
+            "Set collateral factor for USDC"
+        );
```

Comment:
As a precaution to the eventual migration to native USDC as [stated](https://www.circle.com/blog/usdc-now-available-natively-on-base#:~:text=Native%20USDC%20is%20officially%20issued,Circle%20Account%20or%20Circle%20APIs.) by Circle, we are not taking any risk-on recommendations for USDbC. Increases to USDbC's collateral factor will not have any significant impact on capital efficiency since WELL incentives have recently been removed from the USDbC market.